workshops/portfolio/README.md

```diff
@@ -248,7 +246,7 @@ side, we see that our original text is still there:
 
 > ![](img/scroll_down.gif)
 
-Ok. We need to make the picture smaller
+Ok. Let's make the picture smaller
```

Comment:
I think that in this use case, it's actually standing for "let us".

Example that's expanded:

> Let us make the picture smaller